contrib/win32/win32compat/fileio.c

```diff
@@ -749,13 +752,98 @@ fileio_fstat(struct w32_io* pio, struct _stat64 *buf)
 	return _fstat64(fd, buf);
 }
 
+wchar_t * fileio_readlink_internal(wchar_t * wpath)
```

Comment:
This project follows openBSD standards.

return type, filename should be on different lines.

Reply:
Addressed in updated commit.